### PR TITLE
Update ActionMailer base documentation [ci skip]

### PR DESCRIPTION
**Pull request by tomkadwill**
### Summary

Improve documentation for ActionMailer Base.

### CODE REVIEW

File: actionmailer/lib/action_mailer/base.rb [hunk 1]
```diff
@@ -163,7 +163,7 @@ module ActionMailer
   #
   # Multipart messages can also be used implicitly because Action Mailer will automatically detect and use
   # multipart templates, where each template is named after the name of the action, followed by the content
-  # type. Each such detected template will be added as a separate part to the message.
+  # type. Each detected template will be added to the message, as a separate part.
```
Comment (vipulnsward) [inline actionmailer/lib/action_mailer/base.rb]: I would retain the 'such' here for continuity.

Comment (tomkadwill) [inline actionmailer/lib/action_mailer/base.rb]: OK, thanks @vipulnsward. Do you want me to revert the whole line? Or change it to:
`Each such detected template will be added to the message, as a separate part.`

I personally think 'such' is unnecessary but happy to put it back in if you'd like

Comment (vipulnsward) [inline actionmailer/lib/action_mailer/base.rb]: Just retain the "such", rest is good. @tomkadwill if we read a line before and this together, they both are linked and should maintain continuity.

Comment (tomkadwill) [inline actionmailer/lib/action_mailer/base.rb]: Fair point, thanks @vipulnsward. I've updated the PR

**Comment by vipulnsward** [comment]
Thanks @tomkadwill !

**Pull request merged**